stack.yaml

```diff
@@ -58,12 +58,22 @@ packages:
 - location: sys/ui
 - location: sys/value
 - location: sys/valexpr
+- location: sys/txs-lib
+- location: sys/txs-lenses
+- location: sys/txs-webserver
 
 # Dependency packages to be pulled from upstream that are not in the resolver
 # (e.g., acme-missiles-0.3)
 extra-deps:
 - git: https://github.com/TorXakis/text-via-sockets.git
   commit: e3228cd0407ec0d7991a544e154ea2def184fcae
+- servant-0.13
+- servant-server-0.13
+- servant-multipart-0.11.1
+- servant-docs-0.11.2
+- servant-swagger-1.1.5
+- http-types-0.12.1
+- text-1.2.3.0
 
```

Comment:
why are these extra-deps needed?
servant-0.13 is in lts 11.1, like text-1.2.3.0.....

Reply:
I'll check if it works without them.

Reply:
These imports were needed to be able to use `hoistServer`. Now we're just using the `Handler` provided by servant. We shoudn't need these anymore, so I'd guess they can be safely removed.